packages/help-center/src/components/help-center-search-results.tsx

```diff
@@ -224,6 +229,19 @@ function HelpSearchResults( {
 			return <Icon icon={ pageIcon } />;
 		};
 
+		const DeveloperBadge = () => {
+			return (
+				<div className="help-center-search-results__content">
+					{ isResultFromDeveloperWordpress( result.link ) && (
+						<span className="help-center-search-results__badge">
```

Comment:
Might it be possible to use the `<Badge type="info">` component here? Alongside saving a bit of code, that would ensure that the colour contrast is accessible.

Reply:
Hi @Aurorum, great! Will use the badge component for sure 👍